src/olympia/conf/dev/settings.py

```diff
@@ -127,12 +127,21 @@
         'content_host': 'https://stable.dev.lcip.org',
         'oauth_host': 'https://oauth-stable.dev.lcip.org/v1',
         'profile_host': 'https://stable.dev.lcip.org/profile/v1',
-        'redirect_url': 'http://localhost:3000/fxa-authenticate',
+        'redirect_url': 'https://addons-dev.allizom.org/api/v3/accounts/authenticate/?config=local', # noqa
```

Comment:
same here, though I suspect, like with `settings.py`, localhost is correct.

Reply:
Fixed this one too after having fetched the client ID on the server.